Linux/jenkins-node/ansible/readme.md

```diff
@@ -8,9 +8,12 @@ New Linux nodes for use on Jenkins are now set up using ansible scripts in this
 - set up a new node on Jenkins (this will give you the secret code)
```

Comment:
Can you give a short explanation how to do this? i.e. do I copy from an existing node?

Reply:
And do you create the new node on the production jenkins, or does it need to be created on the staging jenkins?

Reply:
In this case, I wouldn't create a new node on either jenkins. I would just connect a new VM to the already existing `isis-cloud-staging`. We can have a different VM pointing at the same node on the a & b instances.